python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -1,197 +1,90 @@
+We consider a random variable :math:`X` of dimension 1 and its unknown quantile :math:`x_{\alpha}` of order :math:`\alpha`.
+We seek to evaluate an upper bound of :math:`x_{\alpha}` with a confidence greater than :math:`\beta`, using a given order statistics.
+
+We denote by :math:`(X_1, \dots, X_\sampleSize)` some independent and identically distributed variables according to :math:`X`.
+
+We denote by :math:`(X_1, \dots, X_\sampleSize)` some independent and identically distributed variables according to :math:`X`.
+
+Let :math:`X_{(k)}` be the :math:`k` -th order statistics of :math:`(X_1, \dots, X_\sampleSize)` which means that
+:math:`X_{(k)}` is the :math:`k` -th maximum of :math:`(X_1, \dots, X_\sampleSize)` for :math:`1 \leq k \leq \sampleSize`. For
+example, :math:`X_{(1)} = \min (X_1, \dots, X_\sampleSize)` is the minimum
+and :math:`X_{(\sampleSize)} = \max (X_1, \dots, X_\sampleSize)` is the maximum. We have:
 
 .. math::
 
-   \begin{aligned}
-       \Prob{ Y^i \leq q_{Y^i}(\alpha)} = \alpha
-     \end{aligned}
+    X_{(1)} \leq X_{(2)} \leq \dots \leq X_{(\sampleSize)}
 
-If we have a sample
-:math:`\left\{ \vect{x}_1,\ldots,\vect{x}_N \right\}` of :math:`N`
-independent samples of the random vector :math:`\vect{X}`,
-:math:`q_{Y^i}(\alpha)` can be estimated as follows:
 
--  the sample :math:`\left\{ \vect{x}_1,\ldots,\vect{x}_N \right\}` of
-   vector :math:`\vect{X}` is first transformed to a sample
-   :math:`\left\{ y^i_1,\ldots,y^i_N \right\}` of the variable
-   :math:`Y^i`, using :math:`\underline{y} = h(\vect{x}_i,\vect{d})`,
+Estimation of the order statistics that will provide an upper bound to :math:`x_{\alpha}`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  the sample :math:`\left\{ y^i_1,\ldots,y^i_N \right\}` is then placed
-   in ascending order, which gives the sample
-   :math:`\left\{ y^{(1)},\ldots,y^{(N)} \right\}`,
+We assume that we have a sample of size :math:`\sampleSize` of :math:`X`, denoted by :math:`(x_1, \dots, x_\sampleSize)`.
+
+Given :math:`\alpha`, :math:`\beta`, and :math:`\sampleSize`, we seek for the smallest :math:`k` such that:
+
+.. math::
+    :label: EqOrderStat
 
--  this empirical estimation of the quantile is then calculated by the
-   formula:
+    \Prob{X_{(k)} \geq x_{\alpha}} \geq \beta
 
-   .. math::
+The order statistics :math:`X_{(k)}` follows the distribution which probability density function and cumulated distribution function
+are defined by:
 
-      \begin{aligned}
-            \widehat{q}_{y^i}(\alpha) = y^{([N\alpha]+1)}
-          \end{aligned}
+.. math::
+    :label: DistOrderStat
 
-where :math:`[N\alpha]` denotes the integral part of
-:math:`N\alpha`.
+    F_{X_{(k)}}(x) & = \sum_{i=k}^{\sampleSize} \binom{\sampleSize}{i}\left[F(x)\right]^i \left[1-F(x)
+    \right]^{\sampleSize-i} \\
+    p_{X_{(k)}}(x) & = (\sampleSize-k+1)\binom{\sampleSize}{k-1}\left[F(x)\right]^{k-1} \left[1-F(x)
+    \right]^{\sampleSize-k} p(x)
```

Comment:
Square brackets are rarely used in mathematics (e.g. the integer part), except with a special meaning. Here, there is no reason to do so.
```suggestion
    F_{X_{(k)}}(x) & = \sum_{i=k}^{\sampleSize} \binom{\sampleSize}{i} \left(F(x)\right)^i \left(1-F(x)
    \right)^{\sampleSize-i} \\
    p_{X_{(k)}}(x) & = (\sampleSize-k+1) \binom{\sampleSize}{k-1} \left(F(x)\right)^{k-1} \left(1-F(x)
    \right)^{\sampleSize-k} p(x)
```

Reply:
ok